packages/block-library/src/columns/edit.js

```diff
@@ -234,7 +234,7 @@ const ColumnsEdit = ( props ) => {
 	}
 
 	return (
-		<Block.div>
+		<Block.div className="has-placeholder">
```

Comment:
Instead of this class, can you target the placeholder component directly? Something like 

```
[data-type="core/columns"] > .components-placeholder {
	margin.... 
}
```

That would also allow us to avoid the :not selector below.

Reply:
I can try this, but the block would need to be aware of the variable. Is that ok?

Reply:
I'm missing some of the context here, so I don't want to give bad advace. In fact I'd defer to your self and @jorgefilipecosta on the technical details — it just seemed like an additional class felt too much. If it isn't, ignore me!